tests/helpers/test-methods.js

```diff
@@ -8,12 +8,25 @@ module.exports = {
   },
   ILogInAs: async ([page, username]) => {
     await page.goto('/user/login');
-    await page.fill('input[name="name"]', username);
-    await page.fill('input[name="pass"]', process.env.APP_SECRET);
-    await page.click('input[value="Log in"]', process.env.APP_SECRET);
+    await expect(page.locator('form.user-login-form')).toBeVisible;
+    await page.locator('input[name="name"]').fill(username);
+    await page.locator('input[name="pass"]').fill(process.env.APP_SECRET);
+    await page.locator('input[value="Log in"]').click();
```

Comment:
Not strictly related to the Drupal Logout, but this seemed a good time to do some refactoring of ILogInAs.

The extra expect() doesn't really add anything, because if we are already logged in, the fill() would also fail.
It's just that the expect() likely generates a better error message.
(Also, I felt I needed to do _something_, while removing the first line from all the following constructions in the tests:
```
    await helpers.IShouldNotBeLoggedIn(page);
    await helpers.ILogInAs([page, 'dru_editor']);
```
)